integration_tests/tests/query_tests/test_queries.py

```diff
@@ -70,6 +79,19 @@
                 "direction": "in",
             },
         ),
+        (
+            "aggregate_network_objects",
+            {
+                "total_network_objects": {
+                    "query_kind": "total_network_objects",
+                    "start_date": "2016-01-01",
+                    "end_date": "2016-01-02",
+                    "aggregation_unit": "admin3",
```

Comment:
Missing a parameter here.

Reply:
Ah but the missing parameter "total_by" is supposed to be optional, with a default of "day" !

Reply:
Ahhh. Looks like default=‘day’ should actually be missing=‘day’ in the totalnetworlobjects schema.

Reply:
Yes, now you mention it, I don't see any references to default value setting in custom_fields.py. I'll try running the tests with:

> class TotalNetworkObjectsSchema(Schema):
>      :::
>    total_by = TotalBy(required=False, default="day")

replaced by:

> total_by = TotalBy(required=False)

But the default must be set somewhere.

Reply:
https://github.com/Flowminder/FlowKit/blob/89851e2c9198e2f2d6378cd810b14cac9e969582/flowmachine/flowmachine/core/server/query_schemas/total_network_objects.py#L20

That line, `default='day'`, should be `missing='day', required=False` - suspect that's what's going on. (I misread the marshmallow docs when I wrote that bit, evidently :))